tsconfig.json

```diff
@@ -1,17 +1,18 @@
 {
-  "include": [ "src/**/*" ],
+  "include": ["./src/**/*"],
+  "exclude": ["./lib/**/*"],
```

Comment:
What is this `lib` folder? It doesn't seem to exist

Reply:
It is folder for generated binary. Could be also called 'build' 🤔